manager/src/main/java/com/redhat/developer/manager/ConnectorsServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.redhat.developer.manager;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.developer.manager.api.exceptions.AlreadyExistingItemException;
+import com.redhat.developer.manager.dao.ConnectorDAO;
+import com.redhat.developer.manager.models.Connector;
+import com.redhat.developer.manager.models.ConnectorStatus;
+import com.redhat.developer.manager.requests.ConnectorRequest;
+
+@ApplicationScoped
+public class ConnectorsServiceImpl implements ConnectorsService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConnectorsServiceImpl.class);
+
+    @Inject
+    ConnectorDAO connectorDAO;
+
+    @Override
+    public Connector createConnector(String customerId, ConnectorRequest connectorRequest) {
+        if (connectorDAO.findByNameAndCustomerId(connectorRequest.getName(), customerId).size() > 0) {
+            throw new AlreadyExistingItemException("Element already present in storage.");
```

Comment:
It would be nice to have a clearer error message e.g. `Connector with name 'foo' already exists for customer with id 'bar'`

Reply:
@r00ta I still think this is a valid comment. Do you disagree?

Reply:
Sorry, I probably closed it by mistake. Fixed now